Feature/task 2

FoxMary opened the pull request.

FoxMary committed:
Task-1

FoxMary committed:
Task-1
Изменила описание ошибок

FoxMary committed:
Task-1
Изменила описание ошибок и объединила методы print

FoxMary committed:
Task-1
Убрала сканнер

FoxMary committed:
Task-1
Исправила print

FoxMary committed:
Task-1
Исправила print

FoxMary committed:
Task-2

iastafyev commented on src/main/java/Main.java:
```diff
@@ -1,5 +1,12 @@
+        try {
+            list.commands();
+        } catch (IOException e) {
+            e.printStackTrace();
```
Не нужно выводить пользователю техническую информацию об ошибке

iastafyev commented on src/main/java/TaskList.java:
```diff
@@ -0,0 +1,104 @@
+    }
+
+    private void add(String description) {
+        taskList.add(new Task(taskList.size() + 1, description));
```
Так будет работать неправильно если комбинировать операции добавления и удаления

iastafyev commented on src/main/java/TaskList.java:
```diff
@@ -0,0 +1,104 @@
+        }
+    }
+
+    private void toggle(String id) {
```
Нет обработки ошибки, что элемента с таким номером не существует

iastafyev commented on src/main/java/TaskList.java:
```diff
@@ -0,0 +1,104 @@
+                coincidence = true;
+            String st = "[ ]";
+            if (t.isStatus()) st = "[x]";
+           out.println(t.getId() + ". " + st + " " + t.getDescription());
```
Дублирование логики печати задачи с методом print.

iastafyev reviewed (commented):
Необходимо сделать merge ветки feature/task-1 в master

FoxMary committed:
Task-2
Исправила:
вывод ошибки в Main
нумерацию задач при добавлении\удалении
обработку ошибки в методе toggle
дублирование логики печати результатов поиска и печати задач

FoxMary committed:
Task-2
Переписала с 0.
